server/cmwell-fts/src/main/scala/cmwell/fts/FTSService.scala

```diff
@@ -300,6 +300,14 @@ class FTSService(config: Config) extends NsSplitter{
     }
   }
 
+  private def isStrSystemField(fieldName: String):Boolean = {
+    fieldName match {
+      case "system.parent" |"system.path" | "system.kind" | "system.uuid" | "system.dc" | "system.quad" |
```

Comment:
system.parent.parent_hierarchy is missing here

Reply:
done